src/frontend/cypress/e2e/chat-list-tests/search-chats.cy.js

```diff
@@ -0,0 +1,75 @@
+describe("search spec", () => {
+  it("filters all", () => {
+    cy.login({ user: "user1", password: "password1" });
```

Comment:
Magari questo (assieme a quelli alle righe 19, 35, 57) si può estrarre in un before each